XiEditor/EditViewController.swift

```diff
@@ -259,6 +259,14 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         editView.inputContext?.discardMarkedText()
         document.sendRpcAsync("select_all", params: [])
     }
+    
+    override func uppercaseWord(_ sender: Any?) {
+        document.sendRpcAsync("uppercase", params: [])
```

Comment:
The way we handle system selectors is that we actually catch them in the `EditView` itself, in the `doCommand` method; there's a map there that translates from Cocoa commands to rpcs. As is neither of these will currently get called.

Reply:
This is... ambiguous. The `Events` enum was a drive-by addition, and hasn't really been used consistently. I think the longer-term goal would be to have a `XiCore` protocol that is implemented by `CoreConnection` (or a thin wrapper) which exposes the API. In any case I don't mind this as is, I don't love the `Events` API particularly, and this is clear.